diff --git a/arch/mips/ath79/dev-usb.c b/arch/mips/ath79/dev-usb.c
index b454115..ab9ac6b 100644
--- a/arch/mips/ath79/dev-usb.c
+++ b/arch/mips/ath79/dev-usb.c
@@ -185,6 +185,9 @@ static void __init ar9xxx_ci_usb_setup(int bus_id, int irq)
 		host_mode = !(ath79_reset_rr(AR934X_RESET_REG_BOOTSTRAP) &
 			      AR934X_BOOTSTRAP_USB_MODE_DEVICE);
 
+	printk(KERN_ERR "ar9xxx_ci_usb_setup force devmode %d\n", host_mode);
+	host_mode = false;
+
 	if (host_mode) {
 		ath79_usb_register("ehci-platform", bus_id,
 				   AR934X_EHCI_BASE, AR934X_EHCI_SIZE,
